pkg/daemon/update.go

```diff
@@ -56,6 +58,8 @@ func (dn *Daemon) update(oldConfig, newConfig *mcfgv1.MachineConfig) error {
 	})
 	if err != nil {
 		return err
+	} else {
+		glog.V(2).Infof("Node successfully drained")
```

Comment:
nit: this doesn't have to be in an else block, since if there is an error we return anyway, so if we get past that we successfully drained. not a big deal but golint complains about it. (it complains about the other instances too, my editor suggests there are 14)

Reply:
Thanks! Will update.

Reply:
Done.